packages/database/src/migrations/20201123061122-AddConfirmedWeeklyDataReportForPSSS-modifies-data.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT_CODE = 'PSSS_CONFIRMED_WEEKLY_REPORT';
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  return insertObject(db, 'report', {
+    id: generateId(),
+    code: REPORT_CODE,
+    config: {
+      fetch: {
+        dataElements: [
+          'PSSS_Confirmed_Sites_Reported',
+          'PSSS_Confirmed_Sites',
+          'PSSS_Confirmed_AFR_Cases',
+          'PSSS_Confirmed_AFR_Alert_Threshold_Crossed',
+          'PSSS_Confirmed_DIA_Cases',
+          'PSSS_Confirmed_DIA_Alert_Threshold_Crossed',
+          'PSSS_Confirmed_ILI_Cases',
+          'PSSS_Confirmed_ILI_Alert_Threshold_Crossed',
+          'PSSS_Confirmed_PF_Cases',
+          'PSSS_Confirmed_PF_Alert_Threshold_Crossed',
+          'PSSS_Confirmed_DLI_Cases',
+          'PSSS_Confirmed_DLI_Alert_Threshold_Crossed',
+        ],
+      },
+      transform: [
+        'keyValueByDataElementName',
+        {
+          transform: 'select',
+          "'period'": "convertToPeriod($row.period, 'WEEK')",
+          '...': '*',
```

Comment:
I can see this happening a lot, so we could consider making it the default behaviour (no harm keeping additional fields, even if they are eventually not needed)